openfe_gromacs/protocols/gromacs_utils/system_creation.py

```diff
@@ -0,0 +1,173 @@
+import os
+import warnings
+
+import gufe
+from openfe.protocols.openmm_utils import charge_generation, system_creation
```

Comment:
I would suggest not calling two different things system_creation, it's too prone to confusion for developers.